goon/browserassets/js/browserOutput.js

```diff
@@ -902,31 +902,25 @@ $(function() {
 	});
 
 	$('#saveLog').click(function(e) {
-		var saved = '';
-
-		if (window.XMLHtpRequest) {
```

Comment:
This was an error but simply fixing this one didn't make the crash go away. I dunno.